reference/employer/openapi.yaml

```diff
@@ -926,6 +932,288 @@ paths:
+  /employer/company/benefits/<benefit_id>:
+    get:
+      summary: List Company Benefit
+      tags:
+        - Employer
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  benefit_id:
+                    type: string
+                  type:
+                    type: string
+                  description:
+                    type: string
+        '404':
+          description: Benefit Not Found
+      operationId: get-company-benefit
+      description: Lists benefit information for a given benefit
+    post:
+      summary: Update Company Benefit
+      operationId: update-company-benefits
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  benefit_id:
+                    type: string
+                  description:
+                    type: string
+        '400':
+          description: Bad Request
+        '404':
+          description: Benefit Not Found
+      description: Updates an existing company-wide benefit
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                description:
+                  type: string
+      tags:
+        - Employer
+  /employer/company/benefits/<benefit_id>/enrolled:
+    get:
+      summary: List Enrolled Individuals
+      tags:
+        - Employer
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  benefit_id:
+                    type: string
+                  individual_ids:
+                    type: array
+                    items:
+                      type: string
+        '404':
+          description: Benefit Not Found
+      operationId: get-company-benefits-enrolled
+      description: |-
+        Lists individuals currently enrolled in a given benefit.
+        Returns empty for `one-time` benefits.
+  /employer/individual/benefits/<benefit_id>:
+    get:
+      summary: List Benefits for Individuals
+      tags:
+        - Employer
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    individual_id:
+                      type: string
+                    employee_deduction:
+                      type: object
+                      properties:
+                        type:
+                          type: string
+                        amount:
+                          type: string
```

Comment:
We seem to use different types throughout. Is there the ability to reference reused "sub objects"? This could be one we could reuse perhaps.